yt_dlp/extractor/boosty.py

```diff
@@ -0,0 +1,209 @@
+                'modified_timestamp': 1694071178,
+                'modified_date': '20230907',
+                'like_count': int,
+                'ext': 'mp4',
+                'duration': 25704,
+                'view_count': int,
+                'thumbnail': r're:^https://i\.mycdn\.me/videoPreview\?',
+            },
+        }, {
+            'info_dict': {
+                'id': '4a3bba32-78c8-422a-9432-2791aff60b42',
+                'title': 'то что не пропустил юта6',
+                'channel': 'Илья Давыдов',
+                'channel_id': '6808257',
+                'timestamp': 1694017040,
+                'upload_date': '20230906',
+                'release_timestamp': 1694017040,
+                'release_date': '20230906',
+                'modified_timestamp': 1694071178,
+                'modified_date': '20230907',
+                'like_count': int,
+                'ext': 'mp4',
+                'duration': 31867,
+                'view_count': int,
+                'thumbnail': r're:^https://i\.mycdn\.me/videoPreview\?',
+            },
+        }],
+    }, {
+        # single external video (youtube)
+        'url': 'https://boosty.to/denischuzhoy/posts/6094a487-bcec-4cf8-a453-43313b463c38',
+        'info_dict': {
+            'id': 'EXelTnve5lY',
+            'title': 'Послание Президента Федеральному Собранию | Класс народа',
+            'upload_date': '20210425',
+            'channel': 'Денис Чужой',
+            'tags': 'count:10',
+            'like_count': int,
+            'ext': 'mp4',
+            'duration': 816,
+            'view_count': int,
+            'thumbnail': r're:^https://i\.ytimg\.com/',
+            'age_limit': 0,
+            'availability': 'public',
+            'categories': list,
+            'channel_follower_count': int,
+            'channel_id': 'UCCzVNbWZfYpBfyofCCUD_0w',
+            'channel_is_verified': bool,
+            'channel_url': r're:^https://www\.youtube\.com/',
+            'comment_count': int,
+            'description': str,
+            'heatmap': 'count:100',
+            'live_status': str,
+            'playable_in_embed': bool,
+            'uploader': str,
+            'uploader_id': str,
+            'uploader_url': r're:^https://www\.youtube\.com/',
+        },
+    }]
+
+    def _real_extract(self, url):
+        user, post_id = self._match_valid_url(url).group('user', 'post_id')
+        post = self._download_json(
+            f'https://api.boosty.to/v1/blog/{user}/post/{post_id}', post_id,
+            note='Downloading post data', errnote='Unable to download post data')
+
+        post_title = post.get('title')
+        if not post_title:
+            self.report_warning('Unable to extract post title. Falling back to parsing html page')
+            webpage = self._download_webpage(url, video_id=post_id)
+            post_title = (self._og_search_title(webpage, fatal=False)
+                          or self._html_extract_title(webpage, fatal=True))
```

Comment:
`title` doesn't need to be fatal (and shouldn't imo), and I don't think we need two warnings
```suggestion
            post_title = self._og_search_title(webpage, default=None) or self._html_extract_title(webpage)
```